authorization/src/lib.rs

```diff
@@ -60,19 +57,19 @@ impl AuthContract {
         e.contract_data().set(DataKey::Admin, admin);
     }
 
-    // Saves data that corresponds to an Identifier, with that Identifiers authorization
-    pub fn save_data(e: Env, auth: Signature, nonce: BigInt, num: BigInt) {
+    /// Save the number for an authenticated [Identifier].
+    pub fn save_num(e: Env, auth: Signature, nonce: BigInt, num: BigInt) {
```

Comment:
It appears we use the terms `data` and `num` to refer to what is being saved. Given the thing being saved is a number, I renamed all the mentions of `data` to `num`.